src/index.ts

```diff
@@ -44,8 +55,8 @@ function setupOpenMRS() {
       },
       {
         id: 'add-patient-action',
-        slot : 'top-nav-actions-slot',
-        load : getAsyncLifecycle(() => import('./add-patient-link'),options),
+        slot: 'top-nav-actions-slot',
+        load: getAsyncLifecycle(() => import('./add-patient-link'), options),
```

Comment:
This array of extensions would also be the place to add our new Edit Patient Button extension.

Reply:
The overflow menu button uses the custom overflowmenu item which is present in the chart-widgets app only. So, I can't add the Edit patient details extension in here.

Reply:
I had this thought to put the customOverflowMenu and customOverflowItem to be put into esm-core/react-utils, so that it can be used somewhere else as well if needed.

Reply:
Yeah. Maybe don't worry about it for now, just leave it in chart-widgets. I need to do some refactoring soon.

Reply:
Sure!